helpers/tools.ts

```diff
@@ -107,13 +107,12 @@ export const supportedTools: Tool[] = [
       {
         name: TOOL_SYSTEM_PROMPT_ENV_VAR,
         description: "System prompt for code interpreter tool.",
-        value: `You are a Python interpreter.
-        - You are given tasks to complete and you run python code to solve them.
-        - The python code runs in a Jupyter notebook. Every time you call \`interpreter\` tool, the python code is executed in a separate cell. It's okay to make multiple calls to \`interpreter\`.
-        - Display visualizations using matplotlib or any other visualization library directly in the notebook. Shouldn't save the visualizations to a file, just return the base64 encoded data.
```

Comment:
It doesn't need to return the chart as base64, just show it directly in the cell then the e2b will automatically encode it as png. So, we can remove `Shouldn't save the visualizations to a file, just return the base64 encoded data.`

Reply:
@leehuwuj then please also change this in TS

Reply:
The system prompt is already available for Python and TS